signac_dashboard/templates/jobs_grid.html

```diff
@@ -42,7 +42,7 @@ <h5 class="subtitle is-5"><a href="{{ url_for('show_job', jobid=job_details.job.
             <div class="card">
                 <div class="card-header">
                     <div class="card-header-title card-header-dashboard">
-                        {% if num_enabled_modules <= 1 and g.jobs | length > 1 %}
+                        {% if num_enabled_modules < 1 and g.jobs | length >= 1 %}
```

Comment:
This essentially means it no modules are enabled and jobs exist. Is this what you want here?